Monika-After-Story/MonikaModDev
# Randchat settings based on aff

Makes randchat naturally slow down as your affection gets lower.

This also changes the default randchat setting to be slower (what was `Less often` is now `Normal`), and the other options have been renamed.
As well, repeat topics now *starts* enabled.

The following table shows the new names for the levels
| Old | New |
|:-:|:-:|
| Often | Very Often |
| Normal | Often |
| Less Often | Normal |
| Occasionally | Less Often |
| Rarely | Occasionally |
| Very Rarely | Rarely |

Additionally, their values have been reversed so the slider works in the correct direction (left being lower, right being higher)

The levels available according to affection are as follows:
| Aff | Available Settings |
|:-:|:-:|
| Broken | Never- Rarely |
| Distressed | Never - Occasionally |
| Upset | Never - Less Often |
| Normal | Never - Normal |
| Happy | Never - Normal |
| Affectionate | Never - Often |
| Enamored | Never - Very Often |
| Love | Never - Very Often |

# Testing:
- Verify no crashes
- Verify randchat is decreased to the level max for the affection level if it drops below the old level's max
- Verify randchat slider gets less options as affection drops
- Verify randchat slider gets more options as affection is gained
- Verify no text clipping on menu
- Verify we default at `Normal`
- Verify we default repeat topics to `True`
